source/includes/_resource.collections.md

```diff
@@ -639,7 +666,8 @@ vhx.collections.update("https://api.vhx.tv/collections/1", {
       </td>
       <td>A set of key/value pairs that you can attach to a collection object. It can be useful for
 storing additional information about the collection in a structured format. Metadata keys must be
-strings. Metadata values can be strings, integers, or arrays. There are a few reserved keys that
+strings. Metadata values can be strings, integers, arrays, or images. An image metadata value
+must must be a url of an image (< 3MB) prefixed with the text <code>image_url:</code>. There are a few reserved keys that
 are auto-generated, which cannot be updated. These reserved keys are: <code>season_number</code></td>
```

Comment:
I know this was already there, but looks like there's just one reserved key listed here, not multiple as suggested?